portfolio/src/main/webapp/script.js

```diff
@@ -42,23 +42,19 @@ function addRandomSong() {
   songContainer.innerText = song;
 }
 
-//Fetches a random message from the server and adds it to the DOM.
-function getRandomMessage() {
-  fetch('/data').then(response => response.text()).then((message) => {
-    document.getElementById('message-container').innerText = message;
-  });
-}
-
 //Fetches the comment from the server and adds it to the DOM.
 function fetchComments() {
-  fetch('/data').then(response => response.json()).then((comment) => {          //may have to change to .then(comments)
-      console.log("fetching comment" + comment);
+  const numComm = document.getElementById('num-choice').value;
+  console.log("numcomm = " + numComm);
+  fetch('/data?num-choice='+numComm).then(response => response.json()).then((comment) => {
```

Comment:
nit: rename this to "comments" to make it clear it's an array.  Or better yet, something like "response", if you decide to do error handling here